.github/workflows/tests.yml

```diff
@@ -49,14 +51,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Setup .NET Core 3.1
+      - name: Setup .NET
         uses: actions/setup-dotnet@v1
         with:
-          dotnet-version: 3.1.x
-      - name: Setup .NET 5.0 # needed for our build steps
-        uses: actions/setup-dotnet@v1
-        with:
-          dotnet-version: 5.0.x
+          dotnet-version: |
+            3.1.x
```

Comment:
I would drop 3.1

Reply:
we test on 'netcoreapp3.1' thats why its required I guess.